src/__tests__/element-queries.js

```diff
@@ -132,6 +134,21 @@ test('query/get element by its title', () => {
   expect(queryByTitle('Delete').id).toEqual('2')
 })
 
+test('query/get element by its value', () => {
+  const {getByValue, queryByValue, getByPlaceholderText} = render(`
+  <div>
+    <input placeholder="name" type="text" id="name"/>
+    <input placeholder="lastname" type="text" id="lastname"/>
+    <input placeholder="email" type="text" id="email"/>
+  </div>
+  `)
+
+  getByPlaceholderText('lastname').setAttribute('value', 'Norris')
```

Comment:
Might be a bit more straightforward to set the `value` initially in the DOM, since the `setAttribute` part is JSDOM's responsibility. What do you think?

Reply:
makes sense, I updated the test.